tutorials/understanding-group/index.md

```diff
@@ -0,0 +1,349 @@
+Like for members management, you need to create a `proposal.json` file that contains the proposal.
+
+A proposal can contain any number of messages defined on the current blockchain.
+
+For this tutorial, you continue with your example of an association. The treasurer, Bob, who wants to send money to a third party to pay the bills, creates a `proposal.json`:
+
+
+```json
+{
+    "group_policy_address": "cosmos1..." , // the $GROUP_POLICY_ADDRESS
+    // array of proto-JSON-encoded sdk.Msgs
+    "messages": [
+        {
+            "@type": "/cosmos.bank.v1beta1.MsgSend",
+            "from_address": "cosmos1...", // the $GROUP_POLICY_ADDRESS
+            "to_address": "cosmos1zyzu35rmctfd2fqnnytthheugqs96qxsne67ad", // a third-party
+            "amount": [
+                {
+                    "denom": "stake",
+                    "amount": "100"
+                }
+            ]
+        }
+    ],
+    "metadata": "costs utilities",
+    "proposers": [ "bobaddr" ] // $BOB
+}
+```
+
+This proposal, if passed, will send 100 `stake` to `cosmos1zyzu35rmctfd2fqnnytthheugqs96qxsne67ad` to pay the bills.
+The tokens will be sent from the decision policy address.
+
+<HighlightBox type="tip">
+
+The decision policy has no funds yet. You can fund it by sending a transaction with `simd tx bank send alice $GROUP_POLICY_ADDRESS 100stake`.
+
+</HighlightBox>
+
+
+```sh
+simd tx group submit-proposal proposal.json --from bob
+```
+
+## View and vote on proposals
+
+Check and verify the proposal of your football association:
+
+```sh
+simd query group proposals-by-group-policy $GROUP_POLICY_ADDRESS
+export PROPOSAL_ID=$(simd query group proposals-by-group-policy $GROUP_POLICY_ADDRESS --output json | jq -r '.proposals[0].id')
+```
+
+You can see that your proposal has been submitted.
+
+Next, have Alice and Bob vote _Yes_ on the proposal and verify that both their votes are tallied:
+
+```sh
+simd tx group vote $PROPOSAL_ID $ALICE VOTE_OPTION_YES "agree"
+simd tx group vote $PROPOSAL_ID $BOB VOTE_OPTION_YES "agree"
+simd query group tally-result $PROPOSAL_ID
+```
+
+Wait for the policy-prescribed 10 minutes, after which your proposal should have passed, as the weighted tally of _Yes_ votes is above the decision policy threshold:
+
+```sh
+simd query group proposal $PROPOSAL_ID
+```
+
+By default proposals are not executed immediately. This is to account for the fact that not everything may be in place to successfully execute the proposal's messages. As you recall, you already funded the group policy. If you had not done it ahead of time, now would have been a good time to fund it.
+	
+Execute the proposal now:
+
+```sh
+simd tx group exec $PROPOSAL_ID --from alice
+```
+
+<HighlightBox type="note">
+
+If there were any errors when executing the proposal messages, none of the messages will be executed and the proposal will be marked as _Failed_.
+
+</HighlightBox>
+
+
+Verify that the proposal has been executed and the tokens have been sent:
```

Comment:
```suggestion
Verify that the proposal has been executed:

simd query group proposal $PROPOSAL_ID

Should return an error:

Error: rpc error: code = Unknown desc = load proposal: not found: unknown request

That is because it has been entirely removed. Confirm that the tokens have been received by the intended recipient:
```